In-house patch to remove unnecessary dependencies from Glance's
requirements files. The specific reasons are as follows:

anyjson			Not applicable

iso8601			Not applicable

kombu			Not applicable

oslo.vmware		Not applicable

posix-ipc		Not applicable

pycrypto		Not applicable to Solaris (M2Crypto used instead)

python-keystoneclient	Not applicable

python-swiftclient	Not applicable

--- glance-2015.1.2/glance.egg-info/requires.txt.~1~	2015-10-13 09:41:01.000000000 -0700
+++ glance-2015.1.2/glance.egg-info/requires.txt	2016-01-24 20:34:20.750596736 -0800
@@ -1,16 +1,12 @@
 pbr!=0.7,<1.0,>=0.6
 greenlet>=0.3.2
 SQLAlchemy<=0.9.99,>=0.9.7
-anyjson>=0.3.3
 eventlet!=0.17.0,>=0.16.1
 PasteDeploy>=1.5.0
 Routes!=2.0,>=1.12.3
 WebOb>=1.2.3
 sqlalchemy-migrate!=0.9.8,<0.10.0,>=0.9.5
 httplib2>=0.7.5
-kombu>=2.5.0
-pycrypto>=2.6
-iso8601>=0.1.9
 oslo.config<1.10.0,>=1.9.3 # Apache-2.0
 oslo.concurrency<1.9.0,>=1.8.2 # Apache-2.0
 oslo.context<0.3.0,>=0.2.0 # Apache-2.0
@@ -19,12 +15,8 @@ stevedore<1.4.0,>=1.3.0 # Apache-2.0
 taskflow<0.8.0,>=0.7.1
 keystonemiddleware<1.6.0,>=1.5.0
 WSME<0.7,>=0.6
-posix-ipc
-python-swiftclient<2.5.0,>=2.2.0
-oslo.vmware<0.12.0,>=0.11.1 # Apache-2.0
 Paste
 jsonschema<3.0.0,>=2.0.0
-python-keystoneclient<1.4.0,>=1.2.0
 pyOpenSSL>=0.11
 six>=1.9.0
 oslo.db<1.8.0,>=1.7.0 # Apache-2.0
--- glance-2015.1.2/requirements.txt.~1~	2015-10-13 09:38:23.000000000 -0700
+++ glance-2015.1.2/requirements.txt	2016-01-24 20:43:04.064101395 -0800
@@ -11,16 +11,12 @@ greenlet>=0.3.2
 
 # < 0.8.0/0.8 does not work, see https://bugs.launchpad.net/bugs/1153983
 SQLAlchemy<=0.9.99,>=0.9.7
-anyjson>=0.3.3
 eventlet!=0.17.0,>=0.16.1
 PasteDeploy>=1.5.0
 Routes!=2.0,>=1.12.3
 WebOb>=1.2.3
 sqlalchemy-migrate!=0.9.8,<0.10.0,>=0.9.5
 httplib2>=0.7.5
-kombu>=2.5.0
-pycrypto>=2.6
-iso8601>=0.1.9
 oslo.config<1.10.0,>=1.9.3 # Apache-2.0
 oslo.concurrency<1.9.0,>=1.8.2 # Apache-2.0
 oslo.context<0.3.0,>=0.2.0 # Apache-2.0
@@ -29,20 +25,11 @@ stevedore<1.4.0,>=1.3.0 # Apache-2.0
 taskflow<0.8.0,>=0.7.1
 keystonemiddleware<1.6.0,>=1.5.0
 WSME<0.7,>=0.6
-# For openstack/common/lockutils
-posix-ipc
-
-# For Swift storage backend.
-python-swiftclient<2.5.0,>=2.2.0
-
-# For VMware storage backed.
-oslo.vmware<0.12.0,>=0.11.1 # Apache-2.0
 
 # For paste.util.template used in keystone.common.template
 Paste
 
 jsonschema<3.0.0,>=2.0.0
-python-keystoneclient<1.4.0,>=1.2.0
 pyOpenSSL>=0.11
 # Required by openstack.common libraries
 six>=1.9.0
